tests/providers/databricks/operators/test_databricks.py

```diff
@@ -1324,7 +1325,7 @@ def test_exec_failure_with_message(self, db_mock_class):
         with pytest.raises(AirflowException) as exc_info:
             op.execute(None)
 
-        assert exc_info.value.args[0].endswith(" Exception: Something went wrong...")
```

Comment:
[`pytest.raises`](https://docs.pytest.org/en/stable/reference/reference.html#pytest.raises) supports match by regexp

I guess this one looks more simple in case if we would like to check only the message, and not exception arguments

```python
        with pytest.raises(AirflowException, match="Exception: Something went wrong"):
            op.execute(None)
```